js/page-specific/phone-directory/phone-directory.js

```diff
@@ -0,0 +1,131 @@
+var jQuery = $.noConflict(true);
+var pageCount = 0;
+
+function __onPrevPage() {
+    for (var i = 1; i <= pageCount; i++) {
+        if (document.getElementById("Page" + i).style.display == "block") {
```

Comment:
I htink you should be able to move this to a variable since this is repeated on line 7

`document.getElementById("Page" + i).style.display`

Reply:
I redid this function by merging with __onNextPage. Let me know if you think it would benefit from further condensing.